docs/src/solvers/ode_solve.md

```diff
@@ -296,6 +296,16 @@ Royal Society, 2011.).
 - `ParsaniKetchesonDeconinck3S205` - 20-stage, fifth order (3S) low-storage scheme, optimised for for the
   spectral difference method applied to wave propagation problems.
 
+#### Extrapolation Methods
+- `AitkenNevillie` - It is a adaptive order, adaptive step size extrapolation method using Aitken - Neville algorithm. Romberg Sequence is used as step-number sequence.
```

Comment:
"An adaptive order, adaptive step size ...". Also, open an issue about expanding the step-number sequence choices.

Reply:
Remove the `it is`